packages/editor/src/core/types/editor.ts

```diff
@@ -78,20 +81,28 @@ export interface IEditorProps {
     suggestions?: () => Promise<IMentionSuggestion[]>;
   };
   onChange?: (json: object, html: string) => void;
+  onTransaction?: () => void;
+  handleEditorReady?: (value: boolean) => void;
+  autofocus?: boolean;
   onEnterKeyPress?: (e?: any) => void;
   placeholder?: string | ((isFocused: boolean, value: string) => string);
   tabIndex?: number;
   value?: string | null;
 }
-
-export type ILiteTextEditor = IEditorProps;
-
+export interface ILiteTextEditor extends IEditorProps {
+  isEnterExtensionEnabled?: boolean;
```

Comment:
We already have a prop to handle disabled extensions, disabledExtensions, make use of that instead of creating a new prop.